org.eclipse.winery.frontends/app/tosca-management/src/app/instance/serviceTemplates/researchObject/files/files.component.html

```diff
@@ -0,0 +1,15 @@
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017 Contributors to the Eclipse Foundation
```

Comment:
If you add new files, please ensure, that the copyright is updated accordingly. Here only 2021.
In case you alter an existing file, please update the copyright to something like 20xx-2021

Reply:
Thanks for the comment, fixed!

Reply:
But this line does not seem to be updated.